### PR TITLE
fix(FX-4085): horizontal offset for popcorn auction banner

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves [FX-4085]

### Description
Inside the auction screen when the auction is a popcorn (cascading) auction, the banner doesn’t seem to have spacing horizontally

### Demo
| Before | After |
| ------------- | ------------- |
| Simulator Screen Shot - iPhone 8 - 2022-07-20 at 13 34 54 | Simulator Screen Shot - iPhone 8 - 2022-07-20 at 13 36 34 |
| before | after |

### QA Test Case(s)

| Test Case | Feature | Environment | Acceptance Criteria | Setup Instructions/Link |
| --------- | :-----: | ----------: | ------------------: | ----------------------: |
|           |         |             |                     |                         |

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-4085]: https://artsyproduct.atlassian.net/browse/FX-4085?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const CascadingEndTimesBanner: React.FC<CascadingEndTimesBannerProps> = (
   const canBeExtended = !!extendedBiddingIntervalMinutes
 
   return (
-    <Flex backgroundColor="blue100" p={2} my={2} ml={-2} mr={-2}>
+    <Flex backgroundColor="blue100" p={2}>
```
Comment (gkartalis) [inline src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx:19]: 👏
**thought(non-blocking):** should we change textAlign to `left`?

File: src/app/Scenes/Artwork/Components/ArtworkTombstone.tsx [hunk 2]
```diff
@@ -218,10 +218,12 @@ export class ArtworkTombstone extends React.Component<
         {!!artwork.isInAuction && !artwork.sale?.isClosed && (
           <>
             {!!artwork.sale?.cascadingEndTimeIntervalMinutes && (
-              <CascadingEndTimesBanner
-                cascadingEndTimeInterval={artwork.sale.cascadingEndTimeIntervalMinutes}
-                extendedBiddingIntervalMinutes={artwork.sale.extendedBiddingIntervalMinutes}
-              />
+              <Flex my={2} ml={-2} mr={-2}>
```
Comment (gkartalis) [inline src/app/Scenes/Artwork/Components/ArtworkTombstone.tsx:221]: **suggestion(non-blocking):** maybe we can change that to something like
```suggestion
              <Flex my={2} mx={-2}>
```

**Review by gkartalis — APPROVED** [review]

File: src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx:19]: It's just a personal preference so feel free to ignore!

**Review by gkartalis — APPROVED** [review]

File: src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/app/Scenes/Artwork/Components/CascadingEndTimesBanner.tsx:19]: I will leave centering the text in the center for now. If it is necessary to make changes, then we will do it in a separate PR

**Review by MrSltun — COMMENTED** [review]
🌟

**Pull request merged**